package.json

```diff
@@ -46,6 +50,7 @@
     "husky": "^6.0.0",
     "jest": "^26.6.3",
     "lint-staged": "^11.0.0",
+    "lodash": "^4.17.21",
```

Comment:
I did think about it.

However, because Theo already uses Lodash, I do not really see a problem here... TBH it's already in `node_modules`, I just explicitly set it here so it's less confusing.

Reply:
Alright I'm fine with it here for now, just have to keep it in mind.